cmd/osbuild-composer/config.go

```diff
@@ -17,14 +17,19 @@ type ComposerConfigFile struct {
 }
 
 type KojiAPIConfig struct {
-	AllowedDomains []string `toml:"allowed_domains"`
-	CA             string   `toml:"ca"`
-	EnableTLS      bool     `toml:"enable_tls"`
-	EnableMTLS     bool     `toml:"enable_mtls"`
-	EnableJWT      bool     `toml:"enable_jwt"`
-	JWTKeysURL     string   `toml:"jwt_keys_url"`
-	JWTKeysCA      string   `toml:"jwt_ca_file"`
-	JWTACLFile     string   `toml:"jwt_acl_file"`
+	AllowedDomains []string  `toml:"allowed_domains"`
+	CA             string    `toml:"ca"`
+	EnableTLS      bool      `toml:"enable_tls"`
+	EnableMTLS     bool      `toml:"enable_mtls"`
+	EnableJWT      bool      `toml:"enable_jwt"`
+	JWTKeysURL     string    `toml:"jwt_keys_url"`
+	JWTKeysCA      string    `toml:"jwt_ca_file"`
+	JWTACLFile     string    `toml:"jwt_acl_file"`
+	AWS            AWSConfig `toml:"aws_config"`
```

Comment:
I'm not sure that it makes sense to have the `AWS` configuration structure in the `KojiAPIConfig`, it seems like a bug to me. As far as I understand the change, the AWS configuration is relevant for the CloudAPI, not KojiAPI. It would probably make more sense to add a new `CloudAPiConfig` structure and have `AWS` moved to it.

Reply:
That is a great idea, thank you <3